test_files/module/main0.d

```diff
@@ -0,0 +1,7 @@
+module modules.main0;
+
+public import modules.include;
```

Comment:
> I want to avoid confusion from the user of translated library

If you look at how Swift imports C/Objective-C libraries, they're trying import the symbols in the most idiomatic way for Swift. For example, many Objective-C method names are renamed to better match Swift names.

> What do you mean by 'package module'? A module that gathers all other sub-modules?

A module with the filename `package.d`, like `std.algorithm` [1].

[1] https://github.com/dlang/phobos/blob/master/std/algorithm/package.d#L183.